algebird-spark/src/main/scala/com/twitter/algebird/spark/AlgebirdRDD.scala

```diff
@@ -1,9 +1,10 @@
-package com.twitter.algebird
+package com.twitter.algebird.spark
```

Comment:
this seems good, but is binary incompatible. Can we just remove this and see if the tests pass?

Reply:
done.